omnibus.gemspec

```diff
@@ -23,9 +23,9 @@ Gem::Specification.new do |gem|
 
   gem.add_dependency 'chef-sugar',       '~> 3.0'
   gem.add_dependency 'cleanroom',        '~> 1.0'
-  gem.add_dependency 'mixlib-shellout',  '~> 1.4'
+  gem.add_dependency 'mixlib-shellout'
   gem.add_dependency 'mixlib-versioning'
-  gem.add_dependency 'ohai',             '~> 7.2'
+  gem.add_dependency 'ohai'
```

Comment:
I wasn't questioning using a later version of mixlib-shellout or ohai but we should still pessimistically lock to the newer Ohai version.

@chef/omnibus-maintainers what are your thoughts on bumping the Ohai version and removing Ruby 1.9 support?